src/scripting/push_check.hpp

```diff
@@ -171,7 +177,7 @@ namespace lua_check_impl
 	}
 
 	//widget
-	//widget not suppored becasue lua_checek returns by value
+	//widget not supported because lua_checek returns by value
```

Comment:
if you are alread fixing this comment: `lua_check for widget is not supported because lua_check returns by value`,

(although now thinking about it, thats not really reason not to support it, we could for example support lua_check for `widget*` or `std::reference_wrapper<widget>`)